lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/RunningWorkers.java

```diff
@@ -38,8 +40,12 @@
 @ThreadSafe
 @DriverSide
 final class RunningWorkers {
+  private static final Logger LOG = Logger.getLogger(RunningWorkers.class.getName());
+
   // RunningWorkers and its locks
   private final HashMap<String, VortexWorkerManager> runningWorkers = new HashMap<>(); // Running workers/tasklets
+  private final Set<Integer> cancelledTasklets = new HashSet<>();
```

Comment:
I'm afraid that this name could be misleading a bit: it sounds this set contains "cancelled" Tasklets, not Tasklets to be cancelled.